lib/roadmap.rb

```diff
@@ -0,0 +1,11 @@
+module Roadmaps
+  def get_roadmap(roadmap_id)
```

Comment:
I see a lot of duplication between these two methods? How would you DRY this up?